usagov/usagov-2021
# USAGOV-1727 - /node/ paths in Published Pages Report (static site generator)

https://cm-jira.usa.gov/browse/USAGOV-1727

## Description
* Fixed a situation where trying to change an alias of a Spanish Basic-Page node would crash the PHP-thread with an error of; `External URLs do not have internal route parameters `
* Fixed a situation where the taxonomy/breadcrumbs (found in the page source-code around the `dataLayer` area) were not always loading the alias-paths for certain nodes
    * this seemed to be an issue with Basic-Page nodes set to the Spanish language

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
1.Edit a node like `/node/1919/edit` and try to change its alias-path. Notice how it crashes on Dev/Test. Now try again after this merge.
1. Run the static-site generator. Before this merge `/node/1919` shows up in the static-site generator. After this marge, its alias path is used instead.

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
4. Find the commit of this pull request.
5. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
